### ISSUE TITLE
Category list

### ISSUE THREAD

**Issue by michzimny** [up2university/docker-moodle#82]
How to reach list of categories in Moodle to select a category I'm interested in? It seems it's hard to find for a regular user. Could we have a list of categories in the front page?

**Comment by michzimny** [comment]
@athird, country managers report it is very important. They do not know now how to reach. without a direct link, the category they manage.

**Comment by michzimny** [comment]
Perhaps the following link could be provided somewhere in the main page?
https://learn.up2university.eu/course/management.php

**Comment by athird** [comment]
We can - ideally, though, we'd only show that to users with the relevant roles, right? (Possible - see, e.g., [here](https://stackoverflow.com/questions/26862933/moodle-2-7-need-to-know-how-to-create-custom-menu-for-different-user-groups))    

Alternatively, there is a setting just to add a list of categories to the front page we can try - I've just manually enabled it on the pedagogy platform. What do you think?

**Comment by michzimny** [comment]
Yes, I think this block is a quick and good solution for that. Thanks to that, people will be able to find the categories they manage.

Persistify this configuration if you can, please.

**Comment by athird** [comment]
OK, it's easy enough to handle. 

I will, there are several changes that need to be persistified, I'll get them put together soon.
